src/Header.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css';
+
+
+class Header extends Component {
+    render() {
+        return (
+            <header>
+                <h1>Dylan's Pokedex</h1>
+                <div className="links">
+                    <NavLink exact to="/">
```

Comment:
Just need to add some active styling to the header NavLinks so the user has a visual indicator of what page they're currently on.